settings.gradle

```diff
@@ -2,6 +2,7 @@ rootProject.name = 'centraldogma'
 
 // Published Java projects
 include 'client:java'
+include 'client:java-spring'
```

Comment:
Think it is better to follow spring-boot module construction manner.
How about renaming this module to client:java-spring-boot-autoconfigure and add another module named client:java-spring-boot-starter that only depends on autoconfigure module?

Reply:
Renamed to `centraldogma-client-spring-boot-autoconfigure` and added `centraldogma-client-spring-boot-starter`.